cnf-certification-test/performance/suite.go

```diff
@@ -79,6 +90,69 @@ var _ = ginkgo.Describe(common.PerformanceTestKey, func() {
 	// Scheduling related tests ends here
 })
 
+func testLimitedUseOfExecProbes(env *provider.TestEnvironment) {
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+	counter := 0
+	for _, put := range env.Pods {
+		for _, cut := range put.Containers {
+			if cut.LivenessProbe != nil && cut.LivenessProbe.Exec != nil {
+				counter++
+				if cut.LivenessProbe.PeriodSeconds <= maxNumberOfExecProbes && cut.LivenessProbe.PeriodSeconds > minExecProbePeriod {
```

Comment:
```suggestion
				if cut.LivenessProbe.PeriodSeconds > minExecProbePeriod {
```

I think you only need to check this condition.  Same with the rest of the following similar sections of code.

Reply:
agree with brandon

Reply:
I agree